CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # CHANGELOG
 
+## `v32.3.0`
+
+### Updated Packages
+
+| Package Name | API Version |
+| -----------: | :---------: |
```

Comment:
Empty table.

Reply:
This empty table is generated by the tool. Do you mean this empty table should be removed?

Reply:
Yeah, looks like it's a bug in the tool.

Reply:
Got it. I will take some investigation this in the tool later